### PR TITLE
Tensorflow support, stage 1: PDEs in loss function

### PR DESCRIPTION
**Pull request by ZongrenZou**

### CODE REVIEW

File: deepxde/model.py [hunk 1]
```diff
@@ -313,11 +325,11 @@ def _test(self, uncertainty):
                 [self.losses, self.net.outputs], feed_dict=feed_dict
             )
         elif backend_name == "tensorflow":
-            outputs = self.net(self.train_state.X_train, training=False)
-            self.train_state.loss_train = self.losses(
-                self.train_state.y_train, outputs
-            ).numpy()
-            self.train_state.y_pred_train = outputs.numpy()
+            y_pred_train, loss_train = self.forward_step(
```
Comment (lululxvi) [inline deepxde/model.py:328]:
```python
y_pred, loss = self.forward_step(
```

File: deepxde/model.py [hunk 2]
```diff
@@ -355,11 +367,11 @@ def _test(self, uncertainty):
                     self.train_state.y_pred_test,
                 ) = self.sess.run([self.losses, self.net.outputs], feed_dict=feed_dict)
             elif backend_name == "tensorflow":
-                outputs = self.net(self.train_state.X_test, training=False)
-                self.train_state.loss_test = self.losses(
-                    self.train_state.y_test, outputs
-                ).numpy()
-                self.train_state.y_pred_test = outputs.numpy()
+                y_pred_test, loss_test = self.forward_step(
```
Comment (lululxvi) [inline deepxde/model.py:370]:
```python
y_pred, loss = self.forward_step(
```
